src/main/java/com/github/vkorobkov/jfixtures/config/ConfigLoader.java

```diff
@@ -5,30 +5,55 @@
 import com.github.vkorobkov.jfixtures.config.yaml.Node;
 import com.github.vkorobkov.jfixtures.util.YmlUtil;
 import lombok.extern.slf4j.Slf4j;
+import lombok.val;
 
 import java.io.IOException;
+import java.nio.file.Files;
 import java.nio.file.Path;
 import java.nio.file.Paths;
+import java.util.Optional;
+import java.util.stream.Stream;
 
 @Slf4j
-public class ConfigLoader {
+public final class ConfigLoader {
     private static final String CONF_YML = ".conf.yml";
-    private final Path path;
+    private static final String CONF_YAML = ".conf.yaml";
 
-    public ConfigLoader(String fixturesRoot) {
-        this.path = Paths.get(fixturesRoot).resolve(CONF_YML);
+    private ConfigLoader() {
     }
 
-    public Root load() {
-        return new Root(loadRootNode());
+    public static Root load(String fixturesRoot) {
+        return new Root(loadRootNode(fixturesRoot));
     }
 
-    private Node loadRootNode() {
-        try {
-            return Node.root(YmlUtil.load(path));
-        } catch (IOException exception) {
-            log.warn("file '.cong.yml' not found, using defaults");
-            return Node.emptyRoot();
+    private static Node loadRootNode(String fixturesRoot) {
+        Optional<Path> configPath = getConfigPath(fixturesRoot);
+
+        if (configPath.isPresent()) {
+            val path = configPath.get();
+            if (YmlUtil.hasTwin(path)) {
+                throw new ConfigLoaderException("Fixture's config exists with both extensions(yaml/yml).");
+            }
+            try {
+                return Node.root(YmlUtil.load(path));
+            } catch (IOException e) {
+                return loadEmptyRoot();
+            }
+        } else {
+            return loadEmptyRoot();
         }
     }
+
+    private static Optional<Path> getConfigPath(String fixturesRoot) {
+        return Stream
+                .of(CONF_YAML, CONF_YML)
+                .map(Paths.get(fixturesRoot)::resolve)
+                .filter(Files::exists)
+                .findFirst();
+    }
+
```

Comment:
Imo we need a separate method for checking that twin does exist:
```java
private static void checkTwin(Path path) {
  if (YmlUtil.hasTwin(path)) {
    throw new ConfigLoaderException("Fixture's config exists with both extensions(yaml/yml).");
  }
}
```

And then we can use this `checkTwin` from  `getConfigPath`:
```java
return Stream
  .of(CONF_YAML, CONF_YML)
  .map(Paths.get(fixturesRoot)::resolve)
  .filter(Files::exists)
  .peek(ConfigLoader::checkTwin)
  .findFirst();
```